samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -340,6 +340,25 @@ See :ref:`ug_bt_fast_pair_provisioning` for details.
    You cannot use the sample without the Fast Pair provisioning data.
    Programming the device with the sample firmware without providing the proper Fast Pair provisioning data results in assertion failure during boot.
 
+Release build
+=============
+
+You can build the sample with a release configuration as a demonstration of how to adapt the default sample configuration for production, using the ``-DFILE_SUFFIX=release`` flag in your build.
+
+The release build reduces code size and RAM usage of the sample by disabling the logging functionality and by performing other optimizations.
+The release build also enables the Link Time Optimization (LTO) configuration - :kconfig:option:`CONFIG_LTO` - to further reduce the code size.
+LTO is an advanced compilation technique that performs optimization across all compiled units of an application at the link stage, rather than within each unit separately.
```

Comment:
```suggestion
The release build reduces the code size and RAM usage of the sample by disabling logging functionality and performing other optimizations. 
Additionally, it enables the Link Time Optimization (LTO) configuration through the :kconfig:option:`CONFIG_LTO` Kconfig option, which further reduces the code size. 
LTO is an advanced compilation technique that optimizes across all compiled units of an application at the link stage, rather than optimizing each unit separately.
```